common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3021,6 +3021,11 @@
           "LATERAL can only be used with subquery and table-valued functions."
         ]
       },
+      "MULTI_PART_CATALOG_NAME" : {
+        "message" : [
+          "<statement> with multiple part catalog name(<catalogName>) is not allowed."
+        ]
+      },
       "MULTI_PART_NAME" : {
         "message" : [
           "<statement> with multiple part function name(<funcName>) is not allowed."
```

Comment:
I'm not sure if it's better to leave this two errors separately or generalized MULTI_PART_NAME to match all use cases

Reply:
I'd like to have a generalized error
```
"<statement> with multiple part name(<name>) is not allowed."
```

Reply:
done